pkg/transport/filtering/object_filtering_map.go

```diff
@@ -43,9 +46,13 @@ type ObjectFilteringMap struct {
 }
 
 func (filteringMap *ObjectFilteringMap) CleanObjectSpecifics(object *unstructured.Unstructured) {
-	filteringFunction, found := filteringMap.gvkToFilteringFunc[object.GetObjectKind().GroupVersionKind()]
+	gvk := object.GetObjectKind().GroupVersionKind()
+	filteringFunction, found := filteringMap.gvkToFilteringFunc[gvk]
 	if !found {
+		klog.V(4).InfoS("No specifics", "gvk", gvk)
 		return // if no filtering function was defined for this gvk, do not clean any field
+	} else {
```

Comment:
why do we need the "else" here ?

Reply:
Now that the unintended logging is removed, there is no longer an "else".

Reply:
My point is that  you don't need it if the if has a return  inside

Reply:
but as it's gone in any case I'm closing. thanks